padertorch/contrib/examples/source_separation/pit/data.py

```diff
@@ -9,16 +9,20 @@
 
 
 def prepare_iterable(
-        db, dataset: str, batch_size, return_keys=None, prefetch=True,
+        db, dataset: str, batch_size, return_keys=None, prefetch=True, shuffle=True
 ):
     audio_keys = ['observation', 'speech_source']
-    iterator = db.get_dataset(dataset)
+    iterable = db.get_dataset(dataset)
```

Comment:
That would be ambiguous because `dataset` is already a `str` that is passed to `prepare_iterable`. Also, I find I natural that the function `prepare_iterable` returns an object called `iterable`. Or do you recommend changing `prepare_iterable`->`prepare_dataset`, `dataset`->`dataset_name` and `iterable`->`dataset`?

Reply:
Yes, that's what I meant. That's more intuitive because it is not clear for everyone that a dataset is an iterable. And I thought that we agreed on the naming `dataset` a while ago.